pkg/kubelet/util/pluginwatcher/plugin_watcher.go

```diff
@@ -178,7 +178,8 @@ func (w *Watcher) init() error {
 func (w *Watcher) traversePluginDir(dir string) error {
 	return w.fs.Walk(dir, func(path string, info os.FileInfo, err error) error {
 		if err != nil {
-			return fmt.Errorf("error accessing path: %s error: %v", path, err)
+			glog.Errorf("error accessing path: %s error: %v", path, err)
```

Comment:
Will this cause all of plugin registration to fail or will it continue to walk?  What kind of traversal ordering does this do?

Reply:
> What kind of traversal ordering does this do?

See [filepath.Walk](https://golang.org/pkg/path/filepath/#Walk): 
> The files are walked in lexical order, which makes the output deterministic but means that for very large directories Walk can be inefficient. Walk does not follow symbolic links.

> Will this cause all of plugin registration to fail or will it continue to walk?

No this will cause the traversePlugin function to ignore directories or files that can't be walked.

Reply:
could we return a failure if it fail to access all dirs and files including root dir?

Reply:
we chat a little offline, i think we only need to return error when root dir access failed.

Reply:
Fixed